File: backend/Entities/Models/Tenant.cs
using System;
using System.Collections.Generic;

namespace Entities.Models;

public partial class Tenant
{
    public Guid Id { get; set; }

    public string Nome { get; set; } = null!;

    public string? Decricao { get; set; }

    public DateTime DataCadastro { get; set; }

    public bool Ativo { get; set; }

    public virtual ICollection<Ciot> Ciots { get; } = new List<Ciot>();

    public virtual ICollection<ClienteCodigo> ClienteCodigos { get; } = new List<ClienteCodigo>();

    public virtual ICollection<Documento1> Documento1s { get; } = new List<Documento1>();

    public virtual ICollection<Dt> Dts { get; } = new List<Dt>();

    public virtual ICollection<Empresa> Empresas { get; } = new List<Empresa>();

    public virtual ICollection<LoteEletronico> LoteEletronicos { get; } = new List<LoteEletronico>();

    public virtual ICollection<Motoristum> Motorista { get; } = new List<Motoristum>();

    public virtual ICollection<Romaneio> Romaneios { get; } = new List<Romaneio>();

    public virtual ICollection<Transportador1> Transportador1s { get; } = new List<Transportador1>();

    public virtual ICollection<Veiculo1> Veiculo1s { get; } = new List<Veiculo1>();
}

File: frontend/src/app/Business/Models/tenant.ts
import { Documento1 } from "./documento1";
import { Empresa } from "./empresa";
import { Romaneio } from "./romaneio";
import { Transportador1 } from "./transportador1";
import { Veiculo1 } from "./veiculo1";

export interface Tenant  {
    id: string;
    nome: string;
    dataCadastro: string;
    ativo: boolean | null;
    documento1s: Documento1[];
    empresas: Empresa[];
    romaneios: Romaneio[];
    transportador1s: Transportador1[];
    veiculo1s: Veiculo1[];
}